Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,53 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            string name;
```

Comment:
Don't declare all variables in the same place. Declare them right next to the place they are used to.
Age- next to getting and parsing age;
Weight- next to getting and parsing weight;
...